content/docs/12-enterprise-version/02-deploying-an-enterprise-cluster.md

```diff
@@ -139,74 +139,123 @@ Use the following steps to install the Palette.
+  | **ECR Registry Private** | Type `y` if the registry is private. Otherwise, type `n`. |
+  | **Use Registry for Images** | Type `y` to use a different registry for images. Type `n` to use the same registry for images. If you are using another registry for images, you will be prompted to enter the registry URL, base path, username, and password. |
 
 <br />
 
-#### VMware vSphere Cluster Configuration
+10. Next, specify the database storage size to allocate for Palette. The default is 20 GB. Refer to the [size guidelines](/vertex/install-palette-vertex#sizeguidelines) for additional information.
+
 
-This information determines where Palette will be deployed in your VMware environment. The Palette CLI will use the provided VMware credentials to retrieve information from your VMware environment and present options for you to select from.
 
+11. The next set of prompts is for the VMware vSphere account information. Enter the information listed in the following table.
 
-|**Parameter**                            | **Description**|
-|-----------------------------------------|----------------|
-|**Datacenter**| The installer retrieves the Datacenter automatically. |
-|**Folder** | Select the folder that contains the VM instance. |
-|**Fault Domains** | Configure one or more fault domains by selecting values for these properties: Cluster, Network (with network connectivity), Resource Pool, and Storage Type (Datastore or VM Storage Policy). Note that when configuring the Network, if you are using a distributed switch, choose the network that contains the switch. |
-|**NTP Servers** | You can provide a list of Network Time Protocol (NTP) servers.  |
-|**SSH Public Keys** | Provide any public SSH keys to access your Palette VMs. This option opens up your system's default text editor. Vi is the default text editor for most Linux distributions. To review basic vi commands, check out the [vi Commands](https://www.cs.colostate.edu/helpdocs/vi.html) reference. |
+  <br />
 
+  #### VMware vSphere Account Information
 
-9. Specify the IP pool configuration. The placement type can be Static or Dynamic Domain Name Server (DDNS). Choosing static placement creates an IP pool from which VMs are assigned IP addresses. Choosing DDNS assigns IP addresses using DNS.
+  |**Parameter**                            | **Description**|
+  |-----------------------------------------|----------------|
+  |**vSphere Endpoint** | vSphere endpoint. Must be a fully qualified domain name (FQDN) or IP address without a scheme - that is, without an IP protocol, such as `https://`. Example: `vcenter.mycompany.com`.|
+  |**vSphere Username** | vSphere account username.|
+  |**vSphere Password**| vSphere account password.|
+  |**Allow Insecure Connection** | Bypasses x509 verification. Type `Y` if using a vSphere instance with self-signed Transport Layer Security (TLS) certificates. Otherwise, type `n`.|
```

Comment:
```suggestion
  |**Allow Insecure Connection** | Bypasses x509 verification. Type `Y` if using a VMware vSphere instance with self-signed Transport Layer Security (TLS) certificates. Otherwise, type `n`.|
```